README.md

```diff
@@ -249,6 +249,8 @@ A curated list of awesome resources related to the Ada and SPARK programming lan
 - [ravenadm](https://github.com/jrmarino/ravenadm) - Administration tool for Ravenports http://www.ravenports.com.
 - [synth](https://github.com/jrmarino/synth) - Next D/Ports build tool for live systems (Alternative for Portmaster and Portupgrade tools).
 - [ada4cmake](https://github.com/mosteo/ada4cmake) - CMake macros for simple gnat project inclusion.
+- [rclada_common](https://github.com/ada-ros/rclada_common) - Common CMake functions for Ada projects.
```

Comment:
I don't know, it's something I came across when searching for cmake Ada stuff. I think you'd have to ask him if it's general enough. If not, stick it together into a ROS specific area or not at all?

Reply:
@mosteo what do you think?

Reply:
Hi! Well, linking any individual packages within the "Ada for ROS2" project does not make much sense, as they're designed to work together. In particular, `rclada_common` is not intended to be used outside of the context of ROS2. It might work, I guess, if one were to dig for the CMake parts only, but it wouldn't be obvious. 

I see that Awesome Ada now links directly to the submodule `rclada`. This is okay and can be left as-is, as the README there is just a redirection to the main repo (https://github.com/ada-ros/ada4ros2), and it's probably less confusing as the names match.

Wrt `ada4cmake`, this is much older and I'm not sure anymore if it should be recommended at all as piece of state-of- the-art Ada tech. In that regard, the intuition that `rclada_common` is more up-to-date is true. Ideally, I should update `ada4cmake` with what I learned doing the ROS2 stuff. Alas, that won't happen any time soon given my agenda :'( But, `ada4cmake` is totally stand-alone, that's true, so it may be worth to keep it.

Reply:
Thank you for the feedback @mosteo. IMO, proper initiatives do not need to be state-of-the-art, else we would wait a long time for many projects. If needs be and you have some time, maybe propose a better "Ada for ROS2" listing structure in the current list as I often cannot fully grasp an author's complete project structure just by skimming the repos.